spock-spring/src/main/java/org/spockframework/spring/mock/DelegatingInterceptor.java

```diff
@@ -51,7 +51,7 @@ public Object intercept(Object target, Method method, Object[] arguments, IRespo
 
     try {
       return method.invoke(delegate, arguments);
-    } catch (IllegalAccessException | InvocationTargetException e) {
+    } catch (ReflectiveOperationException e) {
```

Comment:
I would favour the previous version of this, because using `ReflectiveOperationException` looks to me like Pokemon exception handling. Instead of catching only the specific exceptions that can be thrown, now all exceptions (even unexpected ones / error condition) are caught.